src/iop/retouch.c

```diff
@@ -4055,6 +4045,10 @@ static void retouch_blur(dt_iop_module_t *self, float *const in, dt_iop_roi_t *c
       else
         image_rgb2lab(img_dest, roi_mask_scaled->width, roi_mask_scaled->height, ch, use_sse);
 
+      dt_ioppr_transform_image_colorspace(self, img_dest, img_dest, roi_mask_scaled->width,
```

Comment:
This has already been done in the previous if()

Reply:
Probably a mistake on my side during a merging/rebasing conflict resolution

Reply:
done